modules/ROOT/pages/tools/neo4j-admin/migrate-configuration.adoc

```diff
@@ -57,29 +58,30 @@ The following example shows how to migrate a legacy configuration file to the cu
 
 [source, shell, subs="attributes+"]
 ----
-bin/neo4j-admin server migrate-configuration --from-path=/path/to/legacy/neo4j-enterprise-4.4.10/conf/ --to-path=/path/to/new/neo4j-enterprise-{neo4j-version-exact}/conf/
+bin/neo4j-admin server migrate-configuration --from-path=/path/to/legacy/neo4j-enterprise-4.4.39/conf/ --to-path=/path/to/new/neo4j-enterprise-2025.01.0/conf/
```

Comment:
Since we don't support migrating past 2 major versions, maybe we should have `5.26.0` instead of `4.4.39`?

Reply:
Ok, I'll change it.

Reply:
even though, it worked perfectly 😄